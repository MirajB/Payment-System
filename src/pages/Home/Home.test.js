import HomePage from './index';
import { render } from '@testing-library/react';

it('renders home page', () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
