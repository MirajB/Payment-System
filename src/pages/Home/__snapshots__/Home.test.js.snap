// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page 1`] = `
<div>
  <div>
    <form
      action="#"
    >
      <label
        for="firstName"
      >
        First Name
      </label>
      <input
        class="text-input"
        name="firstName"
        placeholder="Enter first name"
        type="text"
        value=""
      />
      <label
        for="lastName"
      >
        Last Name
      </label>
      <input
        class="text-input"
        name="lastName"
        placeholder="Enter last name"
        type="text"
        value=""
      />
      <label
        for="email"
      >
        Email Address
      </label>
      <input
        class="text-input"
        name="email"
        placeholder="Enter email id"
        type="email"
        value=""
      />
      <label
        for="mobile"
      >
        Mobile No
      </label>
      <input
        class="text-input"
        name="mobile"
        placeholder="Enter mobile No"
        type="number"
        value=""
      />
      <label
        for="emi"
      >
        EMI
      </label>
      <input
        class="text-input"
        name="emi"
        placeholder="Enter EMI to be paid"
        type="number"
        value=""
      />
      <button
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
